src/component/App.js

```diff
@@ -3,21 +3,33 @@ import React, { Component } from 'react';
 import '../scss/index.scss';
 import movieData from '../movieData';
 import Movies from './Movies';
+import MovieInfo from './MovieInfo';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData.movies
+      movies: movieData.movies,
+      selectedMovie: ""
```

Comment:
Is the selectedMovie going to be a string data type? Looking down, it seems like it eventually is an object. Any reasoning for having it start as an empty string?